src/test/java/seedu/address/model/person/FriendNameTest.java

```diff
@@ -37,4 +37,16 @@ public void isValidName() {
         assertTrue(FriendName.isValidFriendName("Capital Tan")); // with capital letters
         assertTrue(FriendName.isValidFriendName("David Roger Jackson Ray Jr 2nd")); // long names
     }
+
+    public void isEqual() {
+        FriendName friendNameOne = new FriendName("Arthur Neo");
+        FriendName friendNameTwo = new FriendName("Bernice Teng");
+
+        assertFalse(friendNameOne.equals(friendNameTwo));
+
+        //Friend name is case insensitive. eg arthur neo is the same as Arthur Neo
+        assertTrue(friendNameOne.equals(new FriendName("arthur neo")));
+
+        assertFalse(friendNameOne == null);
```

Comment:
Any reason for this? Was `.equals` intended?

Reply:
thanks for catching! changed it to `.equals` :)